server/setup.py

```diff
@@ -4,7 +4,7 @@
 CHANGES = ""
 
 requires = [
-    "pyramid==1.10",
+    "pyramid==2.0",
```

Comment:
Better without version so I will able to catch with DEV a problem related to a new version. 
```python
requires = [
    "pyramid",
```